SCClassLibrary/Common/Control/Server.sc

```diff
@@ -884,13 +885,36 @@ Server {
 				this.quit;
 				this.boot;
 			}, {
-				this.bootServerApp({
-					if(startAliveThread) { statusWatcher.startAliveThread }
-				})
+				this.waitForPidRelease {
+					this.bootServerApp({
+						if(startAliveThread) { statusWatcher.startAliveThread }
+					})
+				};
 			}, 0.25);
 		}
 	}
 
+	waitForPidRelease { |onComplete, onFailure, timeout = 1|
```

Comment:
this should be a private method (and hidden in documentation), right?

Reply:
yes, I think so, too.